README.md

```diff
@@ -67,10 +57,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
+- Hat tip to Microverse for providng this template 
 - Inspiration
 - etc
```

Comment:
- [ ] Good job 👏   on customizing the Microverse README however, you need also to put your own information in this `Acknowledgments` section